bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,20 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.6.2
+************
+
+* TLS session cache is again disabled by default due to missing support in modem firmware versions older than 1.1.1 included.
+* :cpp:func:`sendto()` will now set errno to ``NRF_EISCONN`` instead ``NRF_EINVAL`` when passing an address if the socket type is ``NRF_SOCK_STREAM``.
+* Calling :cpp:func:`connect()` on a already connected socket will now properly return ``NRF_EISCONN`` instead of ``NRF_EBADF``.
+* Sockets with family ``NRF_AF_LTE`` must now be created with type ``NRF_SOCK_DGRAM``.
+* Setting a timeout :cpp:func:`recv()` larger than the maximum supported value will now properly return ``NRF_EDOM`` instead ``NRF_EINVAL``.
```

Comment:
```suggestion
* Setting the timeout :cpp:func:`recv()` to a larger than the maximum supported value now properly returns ``NRF_EDOM`` instead of ``NRF_EINVAL``.
```

Reply:
I'll also add an 'in' after word 'timeout'.
Setting the timeout **in** :cpp:func:`recv()` to a larger than the maximum supported value now properly returns ``NRF_EDOM`` instead of ``NRF_EINVAL``.

Reply:
yes, thanks